.github/CODEOWNERS

```
@@ -13,5 +13,7 @@ src/lib/snyk-test/run-iac-test.ts @snyk/cloudconfig
 test/acceptance/cli-test/cli-test.iac-k8s.spec.ts @snyk/cloudconfig
 test/acceptance/cli-test/cli-test.iac-k8s.utils.ts @snyk/cloudconfig
 test/fixtures/iac-terraform/* @snyk/cloudconfig
+test/fixtures/iac-k8s/* @snyk/cloudconfig
```

Small one please
```suggestion
test/fixtures/iac/* @snyk/cloudconfig
```
And move k8s and terraform accordingly

---

@p0tr3c  - pay attention it will require to update the paths in the tests too
https://github.com/snyk/snyk/pull/1421/files#diff-1656000ee8951f40df674f6d329bcbc3R53